packages/web/src/ContentOptionsBuilder.ts

```diff
@@ -62,8 +62,8 @@ class ContentOptionsBuilder {
     return this
   }
 
-  preview() {
-    this.options.preview = true
```

Comment:
`ContentOptionsBuilder.ts` could be in a common dependency `core` to avoid duplicating code

Reply:
Makes sense. Definitely one of the advantages of having all the JS packages in one monorepo.